src/components/tl/mlx5/tl_mlx5_dm.c

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "tl_mlx5_dm.h"
+
+static ucc_status_t ucc_tl_mlx5_dm_chunk_alloc(ucc_mpool_t *mp, //NOLINT
```

Comment:
any reason not to use ucc_mpool_hugetlb_malloc?

Reply:
changed as you suggest